driver-core/src/main/com/mongodb/connection/CommandProtocolImpl.java

```diff
@@ -67,27 +63,12 @@
 
     @Override
     public T execute(final InternalConnection connection) {
-        if (LOGGER.isDebugEnabled()) {
```

Comment:
Logging of command has moved to InternalStreamConnection, which has more information (like the requestId), and can log a truncated JSON representation of the command document without having to double decode.

Reply:
No, logging is not now done via command listener, just done in the same place as command listener since we now are logging basically everything that's available via command listener (which we can't do here).

Reply:
No, it's just handled by the same class that generates the command events.